frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1670,13 +1671,14 @@ end
 
 function ReaderHighlight:editHighlightStyle(page, i)
     local item = self.view.highlight.saved[page][i]
+    local save_document = self.ui.paging and G_reader_settings:readSetting("save_document") ~= "disable"
```

Comment:
No issue with ui.paging documents that don't support saving (pics, djvu) I assume ?

Reply:
It will be nope for them
https://github.com/koreader/koreader/blob/3af268dd7af8050597ad1656ffd0f596e89bf2d4/frontend/document/document.lua#L534-L540